apps/web/src/utils/usernames.ts

```diff
@@ -383,7 +383,7 @@ export function getChainForBasename(username: BaseName): Chain {
 }
 
 // Assume domainless name to .base.eth
-export async function formatDefaultUsername(username: BaseName) {
+export async function formatDefaultUsername(username: string | BaseName) {
```

Comment:
I believe this change is okay, because this function actually anticipates that sometimes it will be called without the `.base.eth` and handles that in the first logic block. 

If this is not okay, then I can do the conversion from name to BaseName on my end.

Reply:
I think you're correct!